### DISCUSSION TITLE
Awesome Oqtane?

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#2616] [General]
inspired by [Awesome Blazor](https://jsakamoto.github.io/awesome-blazor-browser) are there enough modules and apps written in Oqtane to build and "awesome" showcase?

**Comment by sbwalker** [comment]
https://builtonblazor.net/ is showcase of Blazor sites and the site is built on Oqtane....

**Reply by qunshen** [reply]
http://www.jiahu365.com/ and http://vgmeta.cloud123.net/ these are build on Oqtane!
